deploy/gcp/variables.tf

```diff
@@ -80,6 +80,21 @@ variable "tikv_instance_type" {}
 
 variable "tidb_instance_type" {}
 
+variable "pd_image_type" {
+  description = "PD image type, avaiable: UBUNTU/COS"
+  default = "COS"
+}
+
+variable "tidb_image_type" {
+  description = "TiDB image type, avaiable: UBUNTU/COS"
+  default = "COS"
+}
+
+variable "tikv_image_type" {
+  description = "TiKV image type, avaiable: UBUNTU/COS"
+  default = "COS"
+}
```

Comment:
These variables should be added to `deploy/modules/gcp/tidb-cluster/variables.tf`

Reply:
OK, I'll fix it.

Reply:
fixed